server.go

```diff
@@ -275,6 +276,15 @@ func NewFromConfig(conf Config) (*Server, error) {
 		}
 	}
 
+	log.WithField("config", conf).Debug("Initialized server")
```

Comment:
This will spew the API key to our logs if we're not careful... would prefer if we did this after we redact (or only log the sfx host)

Reply:
Oops! Ty, that was an error in a rebase. Fixed up!